pipeline_tools/get_analysis_workflow_metadata.py

```diff
@@ -25,34 +24,24 @@ def get_analysis_workflow_id(analysis_output_path):
     return workflow_id
 
 
-def get_auth(credentials_file=None):
-    """Parse cromwell username and password from credentials file.
-
-    Args:
-        credentials_file (str): Path to the file containing cromwell authentication credentials.
-
-    Returns:
-        requests.auth.HTTPBasicAuth: An object to be used for cromwell requests.
-    """
-    credentials_file = credentials_file or '/cromwell-metadata/cromwell_credentials.txt'
-    with open(credentials_file) as f:
-        credentials = f.read().split()
-    user = credentials[0]
-    password = credentials[1]
-    return HTTPBasicAuth(user, password)
+def get_headers():
```

Comment:
nit: Please add docstrings for this function :octocat:

Reply:
nit: give a more descriptive name to this function, how about `get_oauth_headers` or `get_auth_headers`?